Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+fastapi = "*"
+uvicorn = "*"
```

Comment:
Do not use `*` as a version option. You should specify the version of the lib explicitly, i. e. `~=1.7`